src/olympia/search/tests/test_commands.py

```diff
@@ -229,3 +237,12 @@ def test_create_workflow_addons(self):
         for addons.
         """
         self._test_workflow('default')
+
+    @mock.patch('olympia.search.management.commands.reindex.ES')
+    def test_reindex_skip_if_exists(self, mock_es):
+        """
+        Test reindex command with skip_if_exists option when index already exists.
+        """
+        mock_es.indices.exists.return_value = True
```

Comment:
Not the right mock - it should be `exists_alias` - why is this test passing ?

Reply:
Looksl ike the condition was met regardless.. I simplified the test. Verifiable now that if you modify the mock for alias_exists, the test reliably fails.